### PR TITLE
S3 receiptlog epoch

### PR DESCRIPTION
**Pull request by LeoHChen**

### CODE REVIEW

File: core/block_validator.go [hunk 1]
```diff
@@ -32,9 +32,6 @@ import (
 	"github.com/harmony-one/harmony/internal/params"
 )
 
-// Beacon chain block 1213181 is a one-off block with empty receipts which is expected to be non-empty.
-const beaconBadBlock = 1213181
```
Comment (denniswon) [inline core/block_validator.go]: can you explain why removed?

**Comment by rlan35** [comment]
Thanks picking this up. Can you add the epoch fork logic in the chain config, the same way as CrossLink or Staking? We should put all forking in the same place. The sharding config should only be used for shard structures from now on.

**Comment by LeoHChen** [comment]
> Thanks picking this up. Can you add the epoch fork logic in the chain config, the same way as CrossLink or Staking? We should put all forking in the same place. The sharding config should only be used for shard structures from now on.

Yes. Just changed.

File: core/block_validator.go (hunk 1 above)
Comment (LeoHChen) [reply core/block_validator.go]: use a new way.

**Pull request merged**